### ISSUE TITLE
CM+ not invoked properly using Ctr-M-Space keybinding

### ISSUE THREAD

**Issue by patso23** [oleg-shilo/CodeMaidPlus#7]
I have installed the most recent CM+ along with CodeMaid, and added the command to the third party setting.

However, if I invoke cleanup using Ctr-M-Space, only CodeMaid cleaning is invoked (i.e. System usings are sorted back down alphabetically instead of at the top)

If I invoke cleanup by right-clicking on a project and use Cleanup Selected Code, CM+ is correctly invoke along with CodeMaid.

Not sure if this is a limitation of CodeMaid not invoking third party commands with the shortcut or an issue with CM+

**Comment by oleg-shilo** [comment]
It does look like the integration of CodeMaid with the third-party command is broken.
It can be because of some changes in the CodeMaid itself or because the command name was not specified correctly (e.g. typo).
You can always check the command name from the VS Options dialog:
![image](https://user-images.githubusercontent.com/16729806/181268410-53d07df1-a829-4e7a-b946-59551cc853eb.png)

**Comment by patso23** [comment]
Yes, I had double checked the command name in the VS options menu, it's set correctly. I suppose this is a CodeMaid issue then. 

Thanks for your response

**Comment by oleg-shilo** [comment]
You may report it to the CodeMaid author. 
I did not succeed in getting my work merged with his extension :)
But he is responsive so if you can show him that it is a bug he will address it.
